tests/test_portal.py

```diff
@@ -152,37 +155,40 @@ def test_invite(self):
         postdata = { 'subject': 'unittest subject',
                 'recipients': 'test_user@yahoo.com test_user@uw.edu',
                 'body': "Ode to joy" }
-        rv = self.client.post('/invite', data=postdata, follow_redirects=True)
-        self.assertTrue("Email Invite Sent" in rv.data)
+        response = self.client.post('/invite', data=postdata,
+                                    follow_redirects=True)
+        assert "Email Invite Sent" in response.get_data(as_text=True)
 
     def test_message_sent(self):
         """Email invites - test view for sent messages"""
         sent_at = datetime.strptime("2000/01/01 12:31:00",
                 "%Y/%m/%d %H:%M:%S")
         message = EmailMessage(subject='a subject', user_id=TEST_USER_ID,
-                sender="testuser@email.com",
-                body=u'Welcome to testing \u2713', sent_at=sent_at,
-                recipients="one@ex1.com two@two.org")
+                               sender="testuser@email.com",
```

Comment:
Generic white space request: if you add a newline immediately after the open paren for these multi line statements, the indent is consistent and gives more room.  See https://www.python.org/dev/peps/pep-0008/#indentation

In other words, avoid `Aligned with opening delimiter` pattern - limits space and requires whole code blocks move on edits to the opening line.

Reply:
Gotcha, makes sense.